proxy_agent/src/main.rs

```diff
@@ -64,7 +64,7 @@ async fn main() {
             service::start_service(shared_state.clone());
             println!("Press Enter to end it.");
             let mut temp = String::new();
-            _ = std::io::stdin().read_line(&mut temp);
+            let _read = std::io::stdin().read_line(&mut temp);
```

Comment:
does this every get used?

Reply:
Not used on purpose, hence added under score.